shared/lib/canSafelyAutoCloseThisPopup.test.ts

```diff
@@ -0,0 +1,63 @@
+import browser from 'webextension-polyfill';
+import { canSafelyAutoCloseThisPopup } from './canSafelyAutoCloseThisPopup';
+
+const it = global.it as unknown as jest.It;
```

Comment:
hm, why is this necessary?  I don' think we don't use this logic anywhere else.

Reply:
Replaced with explicit type reference at top of file.